package.json

```diff
@@ -26,6 +26,8 @@
     "bootstrap": "^4.6.0",
     "cheerio": "^1.0.0-rc.10",
     "classnames": "^2.3.1",
+    "crypto": "npm:crypto-browserify",
```

Comment:
do we still need these changes?

Reply:
Removed in https://github.com/isomerpages/isomercms-frontend/pull/1316/commits/a8d740317006ec36f098d92030b3f66ec4c4b37e!